srcs/layouts/cyrl_lynyertz_sr.xml

```diff
@@ -53,4 +73,4 @@
     <key key0="ђ" ne=":" sw="."/>
     <key width="1.5" key0="backspace" ne="delete"/>
   </row>
-</keyboard>
\ No newline at end of file
+</keyboard>
```

Comment:
This is due to the line ending in a newline character (which is the standard), while the previous version didn't.